modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -328,14 +328,15 @@ public function enqueue_scripts() {
 	 * Enqueue WP.com block editor common styles.
 	 */
 	public function enqueue_styles() {
-		// Enqueue only for the block editor in WP Admin.
-		global $pagenow;
-		if ( is_admin() && ! in_array( $pagenow, array( 'post.php', 'post-new.php' ), true ) ) {
+		// On the front-end, add Justify styles inline when Gutenberg styles are also loaded.
+		if ( ! is_admin() ) {
+			wp_add_inline_style( 'wp-block-library', '.has-text-align-justify{text-align:justify;}' );
```

Comment:
Note that `wp-block-library` is not enqueued on the frontend, so we are keeping the same network requests count with this change and also increasing the file size that will be downloaded.

Reply:
See https://github.com/Automattic/jetpack/pull/14233#issuecomment-566209714 (unless there's something _I_ really don't understand 🙃 ).